### ISSUE TITLE
Unable to load sample data into dotnet core app when running in debugger but runs fine in terminal

### ISSUE THREAD

**Issue by chrisjmccrum** [OmniSharp/omnisharp-vscode#908]
## Environment data
`dotnet --info` output:
.NET Command Line Tools (1.0.0-preview2-003131)

Product Information:
 Version:            1.0.0-preview2-003131
 Commit SHA-1 hash:  635cf40e58

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 RID:         osx.10.11-x64

VS Code version: 1.7.1
C# Extension version: 1.4.1

## Steps to reproduce
Cross posted on stackoverflow: http://stackoverflow.com/questions/40533821/unable-to-load-sample-data-into-dotnet-core-app-when-running-in-debugger-but-run
Try and load local project file for data parsing             
`XDocument sampleData = XDocument.Load(@"seed/Import_sampleData.xml");`

## Expected  behavior
Load file without error. Currently when the project is ran from the terminal using dotnet run the file is loaded correctly without error.

## Actual behavior
File Exception error is returned when using the debugger to run the project
`Unhandled Exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.IO.FileNotFoundException: Could not find file '/Users/Chris/Google Drive/Repos/project/project.api/seed/Import_sampleData.xml'.`

**Comment by gregg-miskelly** [comment]
@chrisjmccrum is it trying to read from the path you are expecting? Does that file exist? If you modify your launch.json to set `"justMyCode":false` do you see any other exceptions?

I am pretty doubtful that this is an issue with the C# extension, but we can try and help you route this to the right place (CoreFX ore CoreCLR).

**Comment by chrisjmccrum** [comment]
nevermind I figured it out... basically I have 2 library projects and then my api project all under one subfolder. So the debugger was starting at the root workspace folder that holds the three projects. I changed cwd to start in my api project since the other projects are pulled into it via dependencies and then it immediately found the xml files in my project.

Thank you for your support nevertheless!!
